astilectron.go

```diff
@@ -168,11 +176,14 @@ func (a *Astilectron) Start() (err error) {
 	}
 
 	// Execute
-	if err = a.execute(); err != nil {
-		err = errors.Wrap(err, "executing failed")
-		return
+	if !a.options.SkipAstilectronSetup {
+		if err = a.execute(); err != nil {
+			return errors.Wrap(err, "executing failed")
+		}
+	} else {
+		synchronousFunc(a.canceller, a, func() {}, "app.event.ready")
```

Comment:
Could you replace `func() {}` with `nil` and in `synchronousFunc` wrap `fn()` with `if fn != nil`?

Reply:
done.